pkg/execute/plugin_executor.go

```diff
@@ -124,7 +136,8 @@ func (e *PluginExecutor) Execute(ctx context.Context, bindings []string, slackSt
 	}
 
 	out := interactive.CoreMessage{
-		Message: resp.Message,
+		Message:  resp.Message,
+		Messages: resp.Messages,
```

Comment:
I treat the `Messeges` property as the experimental one as you can read in the comment, it's only SocketSlack feature, meaning I don't want to remove it and also don't want to update all our plugins:
```go
		// Messages holds a collection of messages that should be dispatched to the user in the context of a given command execution.
		// To avoid spamming, you can specify max 15 messages.
		// Limitations:
		//   - It's available only for SocketSlack. In the future, it may be adopted across other platforms.
		//   - Message filtering is not available.
		Messages []api.Message
```